charts/zeebe-benchmark/test/golden/c8-core-configmap.golden.yaml

```diff
@@ -117,8 +116,20 @@ data:
           enabled: true
 
       security:
+        authentication:
+          method: "basic"
+          basic:
+            allowUnauthenticatedApiAccess: false
```

Comment:
:x: This might be an issue for our benchmarks

Can this be disabled?

@megglos ?

Reply:
this is actually outdated config that has no effect anymore on 8.8.0-SNAPSHOT, we probably need to update https://github.com/camunda/zeebe-benchmark-helm/blob/main/charts/zeebe-benchmark/Chart.yaml#L13 ?

new config needed to have unprotected api and no authorizations would be
```
global:
  security:
    authentication:
      unprotectedApi: true
    authorizations:
      enabled: false
```

Reply:
As a follow-up I think it would be worth enabling authentication and authorization in benchmarks indeed though. As they have an overhead and are likely always active in any serious production environment.

Reply:
> As a follow-up I think it would be worth enabling authentication and authorization in benchmarks indeed though. As they have an overhead and are likely always active in any serious production environment. Wdyt?

Sure but first I would like to get the benchmarks running again.